README.md

```diff
@@ -79,44 +81,89 @@ ARGUMENTS
-  name	Name of your plugin. Needs to start with plugin-.
+  name	Name of your plugin. Needs to start with plugin-. This command needs to be invoked inside a plugin directory.
+```
+
+_See code: [src/commands/flex/plugins/create.ts](https://github.com/twilio-labs/plugin-flex/blob/v1.0.0-next.0/src/commands/flex/plugins/create.ts)_
+
+## `twilio flex:plugins:create-configuration`
+
+Creates a Flex Plugin Configuration. This command needs to be invoked inside a plugin directory.
+
+```
+USAGE
+  $ twilio flex:plugins:create-configuration
+
+OPTIONS
+  --description=description  The configuration description
+  --major
+  --minor
+  --new
+  --patch
+
+  --plugin=plugin            (required) The plugin to install, formatted as pluginName@version. Use additional --plugin
+                             to provide other plugins to install
+
+  --version=version
 ```
 
-_See code: [src/commands/flex/plugins/create.js](https://github.com/twilio-labs/plugin-flex/blob/v0.1.6/src/commands/flex/plugins/create.js)_
+_See code: [src/commands/flex/plugins/create-configuration.ts](https://github.com/twilio-labs/plugin-flex/blob/v1.0.0-next.0/src/commands/flex/plugins/create-configuration.ts)_
 
 ## `twilio flex:plugins:deploy`
 
-Builds and deploys your Flex plugin to Twilio Assets. This command needs to be invoked inside a plugin directory.
+Builds and deploys Flex plugin to Twilio Assets. This command needs to be invoked inside a plugin directory.
 
 ```
 USAGE
   $ twilio flex:plugins:deploy
 
 OPTIONS
-  -l=(debug|info|warn|error|none)  [default: info] Level of logging messages.
-  -o=(columns|json|tsv)            [default: columns] Format of command output.
-  -p, --profile=profile            Shorthand identifier for your profile.
+  --changelog=changelog
+  --major
+  --minor
+  --patch
+  --version=version
+```
+
+_See code: [src/commands/flex/plugins/deploy.ts](https://github.com/twilio-labs/plugin-flex/blob/v1.0.0-next.0/src/commands/flex/plugins/deploy.ts)_
+
+## `twilio flex:plugins:release`
+
+Creates a Flex Plugin Release. This command needs to be invoked inside a plugin directory.
+
+```
+USAGE
+  $ twilio flex:plugins:release
+
+OPTIONS
+  --description=description  The configuration description
+  --major
+  --minor
+  --new
+  --patch
+
+  --plugin=plugin            (required) The plugin to install, formatted as pluginName@version. Use additional --plugin
+                             to provide other plugins to install
+
+  --version=version
 ```
 
-_See code: [src/commands/flex/plugins/deploy.js](https://github.com/twilio-labs/plugin-flex/blob/v0.1.6/src/commands/flex/plugins/deploy.js)_
+_See code: [src/commands/flex/plugins/release.ts](https://github.com/twilio-labs/plugin-flex/blob/v1.0.0-next.0/src/commands/flex/plugins/release.ts)_
 
 ## `twilio flex:plugins:remove`
 
-Removes your Flex plugin. This command needs to be invoked inside a plugin directory.
+Removes the Flex plugin. This command needs to be invoked inside a plugin directory.
```

Comment:
can we be consistent with all of the `the Flex plugin/your Flex plugin/Flex plugin`?